<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UKrcmdw</class>
 <widget class="QWidget" name="UKrcmdw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>176</width>
    <height>88</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>54</y>
     <width>62</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
  </widget>
  <widget class="QLabel" name="softIcon">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>20</y>
     <width>48</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="softName">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>16</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="softDescr">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>34</y>
     <width>110</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDetail">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>35</y>
     <width>40</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
